swiftnav_console/main.py

```diff
@@ -474,6 +475,16 @@ def _process_message_buffer(self, buffer):
             data[Keys.SOLID_CONNECTION] = m.statusBarStatus.solidConnection
             data[Keys.TITLE] = m.statusBarStatus.title
             data[Keys.ANTENNA_STATUS] = m.statusBarStatus.antennaStatus
+
+            up = m.statusBarStatus.ntripUpload
+            down = m.statusBarStatus.ntripDownload
+            connected = m.statusBarStatus.ntripConnected
+            if connected:
+                data[Keys.NTRIP_DISPLAY] = f"{up}B/{down}B"
```

Comment:
Should confirm with @switanis but I believe this should be showing bytes per second so that you can tell at a glance if you're getting data from the NTRIP server.

Reply:
Right now it should send the bytes per second,
How should we format this? @switanis 
"Ntrip: 0.0Bps Up/0.0Bps Down" ?